frigate/log.py

```diff
@@ -1,71 +1,70 @@
-# adapted from https://medium.com/@jonathonbao/python3-logging-with-multiprocessing-f51f460b8778
+import atexit
 import logging
 import multiprocessing as mp
 import os
-import queue
-import signal
 import threading
 from collections import deque
-from logging import handlers
-from multiprocessing import Queue
-from types import FrameType
+from contextlib import AbstractContextManager, ContextDecorator
+from logging.handlers import QueueHandler, QueueListener
+from types import TracebackType
 from typing import Deque, Optional
 
-from setproctitle import setproctitle
+from typing_extensions import Self
 
 from frigate.util.builtin import clean_camera_user_pass
 
+LOG_HANDLER = logging.StreamHandler()
+LOG_HANDLER.setFormatter(
+    logging.Formatter(
+        "[%(asctime)s] %(name)-30s %(levelname)-8s: %(message)s",
+        "%Y-%m-%d %H:%M:%S",
+    )
+)
 
-def listener_configurer() -> None:
-    root = logging.getLogger()
+# TODO: Investigate the route cause of this message.
+LOG_HANDLER.addFilter(
+    lambda record: not record.msg.startswith("You are using a scalar distance function")
+)
```

Comment:
```suggestion
LOG_HANDLER.addFilter(
    lambda record: not record.msg.startswith("You are using a scalar distance function")
)
```

Reply:
We know the cause, it is due to old data that caused users to think something was wrong when it wasn't, so the log is filtered out 👍